VENTokenField/VENTokenField.h

```diff
@@ -40,6 +42,7 @@ NS_ASSUME_NONNULL_BEGIN
 - (NSUInteger)numberOfTokensInTokenField:(VENTokenField *)tokenField;
 - (NSString *)tokenFieldCollapsedText:(VENTokenField *)tokenField;
 - (UIColor *)tokenField:(VENTokenField *)tokenField colorSchemeForTokenAtIndex:(NSUInteger)index;
+- (Class)tokenViewForTokenField:(VENTokenField *)tokenField;
```

Comment:
Yes the idea is to supply a subclass. Very similar to how you can supply a subclass to of a UITableViewCell to a UITableViewCell.